src/main/java/org/jenkinsci/test/acceptance/update_center/MockUpdateCenter.java

```diff
@@ -84,20 +84,21 @@ public class MockUpdateCenter implements AutoCleaned {
 
     private HttpServer server;
 
-    public void ensureRunning(Jenkins jenkins) {
+    public boolean ensureRunning(Jenkins jenkins) {
         if (original != null) {
-            return;
+            return false;
         }
         JsonNode ucNode = new UpdateCenter(jenkins).getJson("tree=sites[url,id]");
         List<String> sites = ucNode.findValuesAsText("url");
         List<String> ids = ucNode.findValuesAsText("id");
+        LOGGER.log(Level.INFO, "update sites: {0}", sites);
```

Comment:
I'll remove it later, just wanted to execute test in CB test selector to see that we are using two sites that prevent MockUpdateCenter to start